bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -48,12 +48,17 @@ spec:
             set -o allexport
             source "/env/${K8S_NODE}"
             set +o allexport
-          fi
+          fi  
+          
           echo "$(date -Iseconds) - starting ovn-controller"
           exec ovn-controller unix:/var/run/openvswitch/db.sock -vfile:off \
             --no-chdir --pidfile=/var/run/ovn/ovn-controller.pid \
+            --syslog-method="{{.OVNPolicyAuditDestination}}" \
+            --log-file=/var/log/ovn/acl-audit-log.log \
+            -vFACILITY:"{{.OVNPolicyAuditSyslogFacility}}" \
             -p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt \
-            -vconsole:"${OVN_LOG_LEVEL}"
+            -vconsole:"${OVN_LOG_LEVEL}" -vconsole:"acl_log:off" \
+            -vfile:"acl_log:${OVN_LOG_LEVEL}" -vsyslog:"acl_log:${OVN_LOG_LEVEL}"
```

Comment:
I don't think this is right - OVN_LOG_LEVEL is user-defined, and could be disabled, theoretically. Shouldn't this be constant?

Reply:
So I think for the `vfile` flag I will fix the level to `info` However the syslog flag I will keep fully configurable since it is off by default anyways

Reply:
So, OVN_LOG_LEVEL is the user-defined level, set by configmap, for what should show up in stdout -> `kubectl get logs`. It has nothing to do with ACL logging.

ACL logging should be at whatever causes the logs to show up. It can probably just be fixed to `info`